app/pb_kits/playbook/pb_user/_user.jsx

```diff
@@ -52,12 +54,25 @@ const User = (props: UserProps) => {
     }
   }
 
+  const print_territory = (territory) => {
+    if (territory != "") {
```

Comment:
I think this needs to be !==  , but it might even need to be the triple equals  (!===) to have strict equality. Not sure if territory will ever be anything other than a string. Thoughts? @thestephenmarshall

Reply:
`!==` will do. What I would also suggest here is to set the default up top in the argument like so:

```jsx
const print_territory = (territory="") => {
```